Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/services/DirectoryInitializationService.java

```diff
@@ -128,6 +157,11 @@ public static boolean areDolphinDirectoriesReady()
         return directoryState == DirectoryInitializationState.DOLPHIN_DIRECTORIES_INITIALIZED;
     }
 
+    public static String getUserDirectory()
+    {
+        return userPath;
```

Comment:
It would be nice if we could throw an exception (or have some other kind of noticeable error) if someone tries to call this function before `directoryState` is `DOLPHIN_DIRECTORIES_INITIALIZED`, so that any buggy code that tries to access the user directory too early (hopefully) will get noticed.